Orchid.Jint/JsCallback.cs

```diff
@@ -64,13 +91,13 @@ public object Apply(object @this, params object[] args)
             }
             catch (JavaScriptException jsError)
             {
-                Log.Warning("Scripting", "[{0}:{1}] {2}", jsError.Location.Source, jsError.LineNumber, jsError.Message);
+                Log.Warning("Scripting", $"[{_errorContext}{jsError.Location.Source}:{jsError.LineNumber}] {jsError.Message}");
                 ExecutionError = jsError;
             }
             catch (Exception exception)
             {
                 // TODO: Most recent js stack trace?
```

Comment:
has this todo been acted on yet or nah?

Reply:
No. Out of scope of this change. I may act on it when I upgrade the jint, but that will have a different debug API